robot-command/src/main/java/org/obolibrary/robot/RenameCommand.java

```diff
@@ -216,12 +254,19 @@ private static char getSeparator(String fileName) throws IOException {
       reader.readNext();
       for (String[] nextLine : reader) {
         lineNum++;
-        if (nextLine.length != 2) {
+        if (nextLine.length < 2 || nextLine.length > 3) {
```

Comment:
Maybe nit-picky, but maybe wouldnt it be safer to determine the number of columns in row 1 and then ensuring any subsequent row to be of the same length? Not sure.